src/components/Footer.jsx

```diff
@@ -18,7 +18,7 @@ const useStyles = makeStyles({
 export default function Footer(props) {
 	const classes = useStyles();
 	const simdeLink = <Link href="https://assos.utc.fr/assos/simde" target="_blank" rel="noopener">SiMDE</Link>;
-	const contactLink = <Link href="mailto:simde@assos.utc.fr">Contact</Link>;
+	const contactLink = <Link href="/contact">Contact</Link>;
```

Comment:
```suggestion
	const contactLink = <Link to="/contact">Contact</Link>;
```
`href` is for external links and `to` for internal page change with [react-router](https://reactrouter.com/web/example/basic)